docs/index.md

```diff
@@ -35,59 +35,42 @@ features:
 
 Welcome to all new visitors!
 
-This project is still very new and the team is still working hard on a number
-of areas of the whole project. That includes the website, package repository and
-GitHub organisation as well as the editor itself.
-
-The below is just some items that we are currently working on that might answer
-any questions you have at this early stage of the project.
-
 ### Downloads and Releases
 
 We are constantly working on the editor and maintain both a rolling release and regular release model for our binaries.
 You can find out more about the difference and download a binary of Pulsar by following [this link](https://pulsar-edit.dev/download.html)
```

Comment:
```suggestion
You can find out more about the differences and download Pulsar on our [downloads page](https://pulsar-edit.dev/download.html)
```
I know it isn't part of what you edited but I'm thinking we should change this (and the line above). I think references to binaries are confusing and not really necessary. It also makes it more generic so that people can find repo install links like deb-get and chocolatey.

Also changes the link text, I believe it is generally considered bad form for accessibility to just say "click here" etc.

Reply:
Agreed on not referencing "binaries" so heavily.

> Also changes the link text, I believe it is generally considered bad form for accessibility to just say "click here" etc.

Quite right, it is better for accessibility. (Besides SEO, I think, to whatever extent that matters. Hopefully none, but it might make bots understand relationships between our pages better somehow? And yeah, that's the point, relationships being clearer is the entire idea re: accessibility.)